build-config/scripts/onie-mk-itb.sh

```diff
@@ -139,6 +141,7 @@ DTB="$(realpath $DTB)"
 			arch = "$ARCH";
 			os = "linux";
 			compression = "none";
+			load = <$FDT_LOAD>;
```

Comment:
What was the tool doing before this change, i.e. what was the FDT default load address?

I worry about breaking existing platforms by hard coding FDT_LOAD to 0x9000_0000.

Perhaps we should set FDT_LOAD to a good value in build-config/arch/$(ARCH).make for the relevant architectures and pass that value all the way through to this script.  Exactly how we treat $(KERNEL_LOAD_ADDRESS) today.

The other benefit of that approach is that a specific machine could easily override that value in it's machine.make.